lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py

```diff
@@ -72,11 +73,11 @@
   wait:
     description:
       - Whether or not to wait for the operation to complete. May be useful when waiting for virtual interfaces
-        to be deleted. May modify the time of waiting with C(wait_timeout).
+        to be deleted. May modify the time of waiting with I(wait_timeout).
```

Comment:
This sentence sounds weird to me, though most of the alternatives I've tried sound almost as weird. How about:

```suggestion
        to be deleted. Set the time to wait with I(wait_timeout).
```

Reply:
It feels wrong using the imperative since wait_timeout has a default.   Updated to ``The time to wait can be controlled by setting I(wait_timeout).``